### PR TITLE
Create Validate SOLANA WOLF GANG

### PR DESCRIPTION
**Pull request by SolanaWolfGang**
# Validate [{SWG}](https://solscan.io/token/EEoKiE2Pv9VnK688GXfU36J4U15d986dd7EHK5f9TaBp})

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: (https://x.com/SolanaWolfGang/status/1721909756347158589?s=20)
- Coingecko/ CMC URL (If available): Still working to get on CoinGecko/CMC

## Validation (Please check off boxes):
- [✔️] The metadata provided in the PR matches what is on-chain (Mandatory)
- [✔️] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [✔️] My change matches the format in the file (no spaces between fields).
- [✔️] My token is already live and trading on Jupiter.
- [✔️] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

File: Validate SOLANA WOLF GANG [hunk 1]
```diff
@@ -0,0 +1 @@
++ Solana Wolf Gang, SWG, EEoKiE2Pv9VnK688GXfU36J4U15d986dd7EHK5f9TaBp, 6, https://res2.yourwebsite.life/res/64488cf568b0b0000d804a1f/65065da8c3a072001667cdbe_optimized_201.webp, true
```
Comment (Leonazirite) [inline Validate SOLANA WOLF GANG:1]: Validate SOLANA WOLF GANG

Comment (SolanaWolfGang) [reply Validate SOLANA WOLF GANG:1]: Mint address: EEoKiE2Pv9VnK688GXfU36J4U15d986dd7EHK5f9TaBp
 Symbol: SWG
 Token Decimals: 6
 Token icon: https://res2.yourwebsite.life/res/64488cf568b0b0000d804a1f/65065da8c3a072001667cdbe_optimized_201.webp
 On-chain metadata matches
 No duplication on Jupiter's strict list

**Comment by Timoon21** [comment]
You need to add your token at the end of validated-tokens.csv file. Dont create a new file.

**Comment by Leonazirite** [comment]
Okay thanks

**Comment by meijiao** [comment]
please reopen when PR is fixed

**Pull request closed without merging**